src/components/Counter.js

```diff
@@ -20,10 +20,10 @@ class Counter extends Component {
         return (
             <p>
                 Clicked: {this.props.count} times
-                <button onClick={() => /* Fill me in */ }>
+                <button onClick={() => /* Fill me in */ this.props.increment()}>
```

Comment:
Very good! We imported these methods at the top of this file from actions. Then we passed them as props to this component in our `connect` below! Great work!